### PR TITLE
Sort repositories in an organization

### PR DESCRIPTION
**Pull request by jglick**
Noticed with @christ66 that this is pretty annoying.

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java [hunk 1]
```diff
@@ -506,14 +510,21 @@ public ListBoxModel doFillRepositoryItems(@AncestorInPath SCMSourceOwner context
...
             } catch (IOException e) {
                 LOGGER.log(Level.SEVERE, e.getMessage());
             }
-            return result;
+            return model(result);
+        }
+        private static ListBoxModel model(Collection<String> items) {
+            ListBoxModel model = new ListBoxModel();
+            for (String item : items) {
+                model.add(item);
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: There is a `ListBoxModel.<init>(Collection)` but it takes `Option`s, not `String`s, and those are not `Comparable`.

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java [hunk 2]
```diff
@@ -506,14 +510,21 @@ public ListBoxModel doFillRepositoryItems(@AncestorInPath SCMSourceOwner context
...
                 }
             } catch (SSLHandshakeException he) {
                 LOGGER.log(Level.SEVERE, he.getMessage());
             } catch (IOException e) {
                 LOGGER.log(Level.SEVERE, e.getMessage());
             }
-            return result;
+            return model(result);
+        }
+        private static ListBoxModel model(Collection<String> items) {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: Would be great to see a self-describing method name

**Comment by andresrc** [comment]
:bee:

**Comment by mslusarczyk** [comment]
:bee:

**Pull request merged**